ExampleApp/Shared/Tools.swift

```diff
@@ -24,29 +24,28 @@ struct PaymentPreset {
 }
 
 
-@objc class Tool : NSObject {
+class Tool : NSObject {
+    static let thailandPaymentAmount: Int64 = PaymentPreset.thailandPreset.paymentAmount
+    static let thailandPaymentCurrency: String = PaymentPreset.thailandPreset.paymentCurrency.code
+    static let thailandAllowedPaymentMethods: [SourceType] = PaymentPreset.thailandPreset.allowedPaymentMethods
     
-    @objc static let thailandPaymentAmount: Int64 = PaymentPreset.thailandPreset.paymentAmount
-    @objc static let thailandPaymentCurrency: String = PaymentPreset.thailandPreset.paymentCurrency.code
-    @objc static let thailandAllowedPaymentMethods: [OMSSourceTypeValue] = PaymentPreset.thailandPreset.allowedPaymentMethods
+    static let japanPaymentAmount: Int64 = PaymentPreset.japanPreset.paymentAmount
+    static let japanPaymentCurrency: String = PaymentPreset.japanPreset.paymentCurrency.code
+    static let japanAllowedPaymentMethods: [SourceType] = PaymentPreset.japanPreset.allowedPaymentMethods
     
-    @objc static let japanPaymentAmount: Int64 = PaymentPreset.japanPreset.paymentAmount
-    @objc static let japanPaymentCurrency: String = PaymentPreset.japanPreset.paymentCurrency.code
-    @objc static let japanAllowedPaymentMethods: [OMSSourceTypeValue] = PaymentPreset.japanPreset.allowedPaymentMethods
+    static let singaporePaymentAmount: Int64 = PaymentPreset.singaporePreset.paymentAmount
+    static let singaporePaymentCurrency: String = PaymentPreset.singaporePreset.paymentCurrency.code
+    static let singaporeAllowedPaymentMethods: [SourceType] = PaymentPreset.singaporePreset.allowedPaymentMethods
     
-    @objc static let singaporePaymentAmount: Int64 = PaymentPreset.singaporePreset.paymentAmount
-    @objc static let singaporePaymentCurrency: String = PaymentPreset.singaporePreset.paymentCurrency.code
-    @objc static let singaporeAllowedPaymentMethods: [OMSSourceTypeValue] = PaymentPreset.singaporePreset.allowedPaymentMethods
-    
-    @objc static func imageWith(size: CGSize, color: UIColor) -> UIImage? {
+    static func imageWith(size: CGSize, color: UIColor) -> UIImage? {
         return Tool.imageWith(size: size, actions: { (context) in
             context.setFillColor(color.cgColor)
             context.fill(CGRect(origin: .zero, size: size))
         })
     }
     
-    @objc static func imageWith(size: CGSize, actions: (CGContext) -> Void) -> UIImage? {
-        if #available(iOS 10.0, *) {
+    static func imageWith(size: CGSize, actions: (CGContext) -> Void) -> UIImage? {
+        if #available(iOS 11.0, *) {
```

Comment:
You set the minimum deployment target to iOS 11 already so I think you can remove this here and everywhere else.

Reply:
Nice catch, removed all of availability detection 🙇